### PR TITLE
WIP layout quickfix for subpages

### PR DESCRIPTION
**Pull request by vis-kid**
Add grid system
Center content
Add better margins for content
WIP Improve typograhical measure
WIP removes Sinatra book iframe

### CODE REVIEW

**Comment by vis-kid** [comment]
@zzak thought it might be a good idea to fix these grid issues early to provide documentation readers and such on smaller devices a better reading experience. With this fix I mostly wanted to tame the text and avoid it bumping against browsers windows and stuff. More tweaks to come …

For now I temporarily removed the Sinatra book iframe from /documentation until we find maybe a better home for it :) Ideas?

File: documentation.markdown [hunk 1]
```diff
@@ -6,12 +6,14 @@ layout: default
...
 =============
 
+<!---
 <div style="float: right; margin: 0  0 20px 20px; clear: both">
 <h3>Book</h3>
 <iframe src="http://cdn.oreillystatic.com/widgets/author/61.html" height="500px" width="200px" scrolling="no" frameborder="0" style="margin-top:10px">
   ...
 </iframe>
 </div>
+-->
```
Comment (zzak) [inline documentation.markdown]: > For now I temporarily removed the Sinatra book iframe from /documentation until we find maybe a better home for it :) Ideas?

@rkh Do you have any objection to this?

**Comment by rkh** [comment]
> @rkh Do you have any objection to this?

:+1: fine by me

**Comment by zzak** [comment]
@vis-kid Let me know when you want a review/merge!

File: css/development/legend.css [hunk 2]
```diff
@@ -30,6 +30,9 @@ body {
 
 /* CONTENT */
 
+/***************************
+****************************
+
```
Comment (kgrz) [inline css/development/legend.css]: this code can be deleted instead of commenting out

File: _sass/application.sass [hunk 3]
```diff
@@ -2,5 +2,6 @@
 @import 'neat/neat'
 
 @import 'fonts'
+@import 'grid-styles'
```
Comment (kgrz) [inline _sass/application.sass]: is this part of some library that we use?

Comment (kgrz) [inline _sass/application.sass]: Ah, ignore that

File: _sass/_grid-styles.sass [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#grid-wrapper
+  +outer-container
```
Comment (kgrz) [inline _sass/_grid-styles.sass]: Are these part of some library that we use? should we include it here?

File: css/development/legend.css (hunk 2 above)
Comment (vis-kid) [inline css/development/legend.css]: @kgrz That was not an oversight :). It’s a WIP, not done with yet.

File: _sass/application.sass (hunk 3 above)
Comment (vis-kid) [inline _sass/application.sass]: 😄

File: _sass/_grid-styles.sass (hunk 4 above)
Comment (vis-kid) [inline _sass/_grid-styles.sass]: It’s part of Bourbon Neat.

**Comment by vis-kid** [comment]
Took care of conflicts and on my end it should be ready to merge :)

**Comment by zzak** [comment]
@vis-kid I merged the typography one, now this needs a rebase ><

**Comment by vis-kid** [comment]
@zzak on it!

**Comment by vis-kid** [comment]
@zzak All green again!

**Comment by vis-kid** [comment]
Nice!! 👍

**Comment by vis-kid** [comment]
@zzak While you are at it, the commit that got rid of “unused JS” actually removed code that handles the menu on smaller devices. Atm, the menu is not working. How should we best deal with that?

**Comment by zzak** [comment]
@vis-kid Send me a patch :)

**Pull request merged**